README.md

```diff
@@ -33,7 +33,7 @@ pip install git+https://github.com/iMicknl/python-tahoma-api.git@master#tahoma_a
 
 - Clone this repository
 - `cd python-tahoma-api`
-- Install the required Python version: `pyenv install 3.8.3`
+- Install the required Python version: `pyenv install`
```

Comment:
`pyenv install` use the version set in .python_version.

Reply:
Is the pre-commit hook not requiring Python 3.8? That is what I remember from my struggles.

Reply:
Nope, it can run using the 3.6 also.